kratos/python/add_utilities_to_python.cpp

```diff
@@ -524,13 +524,31 @@ void AddUtilitiesToPython(pybind11::module& m)
         .def("UpdateSearchDatabase", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabase)
         .def("UpdateSearchDatabaseAssignedSize", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabaseAssignedSize)
         .def("FindPointOnMesh", &BinBasedFastPointLocator < 2 > ::FindPointOnMeshSimplified)
+        .def("FindPointOnMesh", [](BinBasedFastPointLocator < 2 >& self, const array_1d<double,3>& coords, const double tol)
+            {
+                Element::Pointer pelem;
+                Vector N;
+                int max_results = 1000;
+                bool found = self.FindPointOnMeshSimplified(coords, N, pelem, max_results, tol);
+                return py::make_tuple(found, N, pelem);
```

Comment:
why you should return the N?
is not the lentgh of result?

Reply:
no, it is the value of the shape functions. you need to know:
1 - if you found an element you are in
2 - the value of the shape functions within the element (if found=true)
3 - the element you are in

Reply:
I see, Thanks